adobe_vipm/flows/benefits.py

```diff
@@ -165,102 +169,3 @@ def update_deployment_agreements_3yc(
             deployment_agreement["id"],
             parameters=parameters_3yc,
         )
-
-
-def resubmit_3yc_commitment_request(mpt_client, is_recommitment=False):
```

Comment:
@jentyk @alephsur  why this code is not used anymore? Extension now doesn't support recommitment requests? why?

Is it some new requirement from Stu?

Reply:
I am not sure, but it wasn't used yet when I joined